libsplinter/src/oauth/rest_api/actix/callback.rs

```diff
@@ -83,15 +90,57 @@ pub fn make_callback_route(
                                     {
                                         Ok(session) => {
                                             match oauth_user_session_store.add_session(session) {
-                                                Ok(_) => HttpResponse::Found()
+                                                Ok(_) => {
+                                                    // Save the user profile data if the biome-profile feature is enabled
+                                                    #[cfg(feature = "biome-profile")]
+                                                    {
+                                                        let profile = Profile::from(&user_info.profile().clone());
+                                                        match user_profile_store.get_profile(profile.user_id()) {
+                                                            Ok(profile) => { match user_profile_store.update_profile(profile) {
+                                                                    Ok(_) =>  {
+                                                                        info!("User profile updated");
```

Comment:
This should probably be `debug` instead of `info`, since it generally won't be important information for admins to see every time a user's profile is updated. Same for the "User profile saved" message below.

Reply:
Fixed